compiler/class-assumptions.cpp

```diff
@@ -492,7 +492,7 @@ static Assumption calc_assumption_for_var(FunctionPtr f, const std::string &var_
 
   if (f->type == FunctionData::func_main || f->type == FunctionData::func_switch) {
     if (var_name == "MC" || var_name == "PMC") {
-      assumption_add_for_var(f, var_name, Assumption(G->get_memcache_class()), f->root);
+      return Assumption(G->get_memcache_class());
```

Comment:
Don't understand this change. Could you explain please?

Reply:
This is not a meaningful change, just a minor style fix I've accidently notices (to make this line be in style of above and below, that also use "return")

Reply:
ok, got it